internal/xds/translator/testdata/out/xds-ir/http-route-with-tlsbundle-mtls.clusters.yaml

```diff
@@ -0,0 +1,31 @@
+- commonLbConfig:
+    localityWeightedLbConfig: {}
+  connectTimeout: 10s
+  dnsLookupFamily: V4_ONLY
+  edsClusterConfig:
+    edsConfig:
+      ads: {}
+      resourceApiVersion: V3
+    serviceName: httproute/envoy-gateway/httproute-btls/rule/0
+  lbPolicy: LEAST_REQUEST
+  name: httproute/envoy-gateway/httproute-btls/rule/0
+  outlierDetection: {}
+  perConnectionBufferLimitBytes: 32768
+  transportSocket:
+    name: envoy.transport_sockets.tls
+    typedConfig:
+      '@type': type.googleapis.com/envoy.extensions.transport_sockets.tls.v3.UpstreamTlsContext
+      allowRenegotiation: true
+      commonTlsContext:
```

Comment:
This example implies that we actually support upstream MTLS through this PR... I think that the GW API currently only means for ca.crt to be provided via `CACertRefs`, so using this to create a `tls.key, tls.crt` and providing these as client auth via the cluster is probably not compliant...

Reply:
removed